tests/reports/outputs/report1.json

```diff
@@ -57769,7 +57769,7 @@
             "comparisons": [
                 {
                     "name": "docker0.ipv4_addresses",
-                    "state": "DIFFERENT",
+                    "state": "INCOMPLETE_DATA",
```

Comment:
are the changes here expected? or should the status only be altered for FQDN?

Reply:
the changes here are expected because fqdn is treated the same as *_addresses, all are considered unique rec names, as far as I understand it.

Reply:
To clarify, with recent work, we are changing behaviour for statuses to be Incomplete for any "N/A" data, and that's not specific to fqdn, it's just one of many places the change was made and needed some extra logic, if I recall correctly.